code/modules/research/techweb/all_nodes.dm

```diff
@@ -117,6 +117,7 @@
 	prereq_ids = list("adv_surgery")
 	design_ids = list("surgery_pacify","surgery_vein_thread","surgery_muscled_veins","surgery_nerve_splice","surgery_nerve_ground","surgery_ligament_hook","surgery_ligament_reinforcement","surgery_cortex_imprint","surgery_cortex_folding","surgery_viral_bond", "surgery_heal_combo_upgrade")
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 5000)
+	discount_experiments = list(/datum/experiment/scanning/random/plants/traits = 2500)
```

Comment:
personal rpeference is that you increase the cost of the node as well so its more likely people will do the experiment

Reply:
(and then make it so that the experiment cuts out a bigger portion of that cost)